test/opf-var.jl

```diff
@@ -207,6 +244,20 @@ end
         end
     end
 
+    @testset "test mi dc opf" begin
+    @testset "5-bus case" begin
+        result = PowerModels._run_strg_mi_opf("../test/data/matpower/case5_strg.m", PowerModels.DCPPowerModel, juniper_solver)
```

Comment:
In this case can use use CBC instead of Juniper?

Reply:
The objective in case5_strg is quadratic, and CBC/GLPK won't solve it